aws-opentelemetry-distro/src/amazon/opentelemetry/distro/patches/_instrumentation_patch.py

```diff
@@ -20,6 +25,40 @@ def apply_instrumentation_patches() -> None:
 
     Where possible, automated testing should be run to catch upstream changes resulting in broken patches
     """
+    if _is_installed("gevent"):
+        try:
+            gevent_patch_module = os.environ.get(AWS_GEVENT_PATCH_MODULES)
+            if gevent_patch_module == "all":
+                # pylint: disable=import-outside-toplevel
+                # Delay import to only occur if monkey patch is needed (e.g. gevent is used to run application).
+                from gevent import monkey
+
+                monkey.patch_all()
+            elif gevent_patch_module == "none":
+                pass
+            else:
+                module_list = [module.strip() for module in gevent_patch_module.split(",")]
+
+                # pylint: disable=import-outside-toplevel
+                # Delay import to only occur if monkey patch is needed (e.g. gevent is used to run application).
+                from gevent import monkey
```

Comment:
Try not to repeat statements for better maintainability. You can rearrange the if-else blocks to only import the module once when `gevent_patch_module` is NOT `none`.